wiki/Help_centre/Account_restrictions/zh.md

```diff
@@ -106,17 +109,19 @@ outdated_since: b830432c4768e242ce32ad592010de960bbefdeb
 
 请您知晓：向这个地址发邮件只能保证对您的申诉的复审或应答。如果您被发现不符合申诉过程的期望，复审不会保证申诉结果的改变。
 
-**我们只会对发送至 [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) 的复审或调查请求进行答复。** 不要试图通过推特，Discord，QQ 等方式联系开发人员（以及 GMT、NAT 等志愿者团队）来绕过这个政策——他们帮不了您。
+**我们只会对发送至 [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) 的复审或调查请求进行答复。**不要试图通过推特，Discord，QQ 等方式联系开发人员（以及 GMT、NAT 等志愿者团队）来绕过这个政策——他们帮不了您。
```

Comment:
should probably specifically mention Twitter as in EN as that's where some people on these volunteer teams have accounts